region/brinstar/green.json

```diff
@@ -2246,13 +2253,54 @@
               ]
             },
             {
-              "id": 4,
+              "id": 6,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    "Morph"
+                  ]
+                },
+                {
+                  "name": "Spikeway Morphless Tunnel Crawl (Left to Right)",
+                  "notable": true,
+                  "requires": [
+                    "canMorphlessTunnelCrawl",
+                    "canTrickyJump",
+                    {"thornHits": 31}
+                  ],
+                  "note": [
+                    "Moving left to right is a bit more tricky to get into the tunnel.",
+                    "Land aiming down approximately 3 pixels inside the leftmost tile.",
+                    "If Samus is too far left she will fall out, too far right and a spike pushes you out, not aiming down puts her stuck in crouch.",
```

Comment:
There's a downback setup I like for both directions, but maybe that would be separate reference material for this strat.  Like Xray helping with the setup.

Reply:
If you want to describe them both, I can add them to the note

Reply:
L->R
Fall off with low speed and start a downback right away.  (Doesn't always work.)

R->L
Stand fully on the left side of the nearby ledge and run off then start and hold a downback to land in position.